### PR TITLE
Show NCP firmware modules in the output of `serial inspect`

### PR DESCRIPTION
**Pull request by sergeuz**
This PR introduces the following changes:

- Show NCP firmware modules in the output of the `serial inspect` command.
- Increase the timeout of the `s` and `i` serial commands sent to the device.

Example output of the `serial inspect` command with an Argon device:
```
Platform: 12 
Modules
  Bootloader module #0 - version 300, main location, 49152 bytes max size
    Integrity: PASS
    Address Range: PASS
    Platform: PASS
    Dependencies: PASS
  System module #1 - version 1002, main location, 671744 bytes max size
    Integrity: PASS
    Address Range: PASS
    Platform: PASS
    Dependencies: PASS
      Bootloader module #0 - version 201
  User module #1 - version 6, main location, 131072 bytes max size
    UUID: 8C3E879F1032133D7F456603A8A52DDFBA1102CF472FFC376CED0D69075FB7C3
    Integrity: PASS
    Address Range: PASS
    Platform: PASS
    Dependencies: PASS
      System module #1 - version 1002
  NCP module #0 - version 5, main location, 1536000 bytes max size
    Integrity: PASS
    Address Range: PASS
    Platform: PASS
    Dependencies: PASS
```

### CODE REVIEW

File: src/cmd/serial.js [hunk 1]
```diff
@@ -39,6 +39,10 @@ const protip = () => {
 
 const timeoutError = 'Serial timed out';
 
+// An LTE device may take up to 18 seconds to power up the modem
+const MODULE_INFO_COMMAND_TIMEOUT = 20000;
+const IDENTIFY_COMMAND_TIMEOUT = 20000;
```
Comment (busticated) [inline src/cmd/serial.js]: did you consider placing these in `./settings.js` ([example](https://github.com/particle-iot/particle-cli/blob/c9f746b5041ad58e788aa42a710a842b61abcb43/settings.js#L46))?

i realize we have a bunch of these scattered throughout this file so what you have is an improvement but i'm curious about your thinking here.

Comment (sergeuz) [reply src/cmd/serial.js]: These constants are only relevant to the `i` and `s` commands of the serial console, and I'd keep them closer to the code that actually works with those commands

**Comment by busticated** [comment]
@sergeuz same note here re: rebasing - https://github.com/particle-iot/particle-cli/pull/481#issuecomment-486732967

is this good to merge?

**Comment by sergeuz** [comment]
> is this good to merge?

@busticated Yep!

**Pull request merged**